example_buffer.py

```diff
@@ -0,0 +1,194 @@
+import tensorflow as tf
+import os
+import random
+from tqdm import tqdm
+import multiprocessing as mp
+import functools
+import itertools
+import rl_loop
+import dual_net
+import subprocess
+from utils import timer
+import time
+import preprocessing
+import argh
+import argparse
+import datetime as dt
```

Comment:
I general sort third party in a separate group https://www.python.org/dev/peps/pep-0008/#imports